tools/test

```diff
@@ -35,6 +35,16 @@ Options:
   --fix       Fix issues found, where possible.
   --all-files
               Run on all files, not only changed files.
+  --platform <ios|android|both|sloppy>
+              Run tests as if on iOS, or Android, or both. The default,
+              "sloppy", takes a shortcut in some suites, so the tests
+              run faster but full coverage of both platforms isn't
+              guaranteed. In particular, while the vast majority of
+              our Jest tests don't depend meaningfully on the platform,
+              we don't yet have a clever way to identify the ones that
+              do. So, "sloppy" just picks one platform to simulate
+              (iOS). Don't use "sloppy" if you want coverage of the
+              other platform.
```

Comment:
I think this could be clearer:

```
              Run tests as if on iOS, or Android, or both. The default,
              "sloppy", takes a shortcut in some suites, so the tests
              run faster but full coverage of both platforms isn't
	      guaranteed. Specifically, "sloppy" will run both Android and iOS
	      native tests, but will run Jest tests on only one platform (iOS).
	      This is usually fine, because the vast majority of our Jest tests
	      don't depend meaningfully on the platform.
```

Reply:
I like it!